src/lib/features/project-environments/environment-service.ts

```diff
@@ -77,8 +76,24 @@ export default class EnvironmentService {
 
     async getProjectEnvironments(
         projectId: string,
-    ): Promise<IProjectEnvironment[]> {
-        return this.environmentStore.getProjectEnvironments(projectId);
+    ): Promise<IProjectsAvailableOnEnvironment[]> {
+        // This function produces an object for every environment, in that object is a boolean
```

Comment:
I'm usually not a fan of explaining what's going on with comments but this behaviour is so surprising to me that I had to reverse engineer it from API behaviour and it'd be cool to save the next person the trouble

Reply:
Maybe something to add to the list of major API changes for next release, but first we'd have to deprecate it in the API in the next major